api/src/main/java/com/codeforcommunity/rest/subrouter/authenticated/ProtectedUserRouter.java

```diff
@@ -63,6 +67,18 @@ private void registerChangeEmail(Router router) {
     changePasswordRoute.handler(this::handleChangeEmailRoute);
   }
 
+  private void registerGetAllUsers(Router router) {
+    Route getAllUsersRoute = router.get("/");
```

Comment:
can I change this to get("")? Therefore the route will be /user?country=<country> instead of /user/?country=>country>. The extra slash in the end is kind of annoying

Reply:
I have no idea. @connernilsen @justinkonecny @liammoyn

Reply:
no you cannot, just tried it

Reply:
Wait does it not allow `/user?country...` when you have the route as `/`?

Reply:
I tried having the router.get("/") be router.get("") and it threw an exception

Reply:
Oh yeah, I guess that makes sense in terms of path semantics. Are you allowed to access `/user?country...` when the method calls `router.get("/")` though?